crates/biome_js_parser/src/syntax/object.rs

```diff
@@ -89,6 +89,12 @@ pub(super) fn parse_object_expression(p: &mut JsParser) -> ParsedSyntax {
 /// An individual object property such as `"a": b` or `5: 6 + 6`.
 fn parse_object_member(p: &mut JsParser) -> ParsedSyntax {
     match p.cur() {
+        // test js object_expr_metavariable_member
```

Comment:
This line should be removed. `// test` lines were part of a very old test framework that has long since been removed.

```suggestion
```